tap_pendo/streams.py

```diff
@@ -378,8 +377,16 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
 
                 # Filtering the visitors based on last updated to improve performance of visitor_history replication
                 if isinstance(parent, Visitors):
+                    if record['metadata'].get('pendo', {'donotprocess': False}).get('donotprocess'):
+                        # If any visitor is set the Do Not Process flag then Pendo will stop collecting events
+                        # from that visitor and will stop displaying guides to that visitor
```

Comment:
Can we update this docstring to explain why we skip those records, eg they will not contain required keys and we can't persist them?